### ISSUE TITLE
Add PR Template

### ISSUE THREAD

**Issue by arshadkazmi42** [arshadkazmi42/squash#18]
Add pull request template.
Create a file name `PULL_REQUEST_TEMPLATE.md` in root directory

and add the following in it

```
Fixes #issuenumber

Check everything which applies
[] I have added the issue number for which this pull request is created.

```

**Comment by ConSou** [comment]
Hello,
I can get this added to the project

**Comment by arshadkazmi42** [comment]
Closes #18 via #25
